ray-operator/config/samples/ray-service.mobilenet.yaml

```diff
@@ -12,6 +12,11 @@ spec:
         runtime_env:
           working_dir: "https://github.com/ray-project/serve_config_examples/archive/b393e77bbd6aba0881e3d94c05f968f05a387b96.zip"
           pip: ["python-multipart==0.0.6"]
+        deployments:
+          - name: ImageClassifier
```

Comment:
How did this YAML even work before without a `deployments` field?

Reply:
The cluster will be deployed on the head Pod, as KubeRay submits the Serve application once the head node is ready. In most cases, workers are not ready.